src/test/java/seleniumTest/BaseTest.java

```diff
@@ -0,0 +1,29 @@
+package seleniumTest;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import yandex.mail.pages.BasePage;
+
+import static java.time.Duration.ofSeconds;
+
+public class BaseTest {
+    protected WebDriver driver;
+
+    @BeforeMethod
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().pageLoadTimeout(ofSeconds(30));
+        driver.manage().timeouts().implicitlyWait(ofSeconds(15));
```

Comment:
I don't understand why I need move 15 seconds to separate value. It is used in BaseTest and InboxPage and I can't to reused this variable.

Reply:
I mean better put it to:
public static final int DEFAULT_TIMEOUT = 15;

because for another person easy to see in the beginning of class var DEFAULT_TIMEOUT, then trying to find it somewhere in the code